sources/helpers/firebase.js

```diff
@@ -0,0 +1,23 @@
+const admin = require('firebase-admin');
```

Comment:
I like the logic/flow in this file. Awesome actually. Works neatly.

Hopefully🙏